@kiva/kv-components/src/vue/KvSideSheet.vue

```diff
@@ -236,22 +273,33 @@ export default {
 			}
 		};
 
-		// Set up ResizeObserver and window resize listener
+		// Set up resize listeners
 		onMounted(() => {
-			updateContentHeight();
+			// Initial measurement after a delay to ensure hydration
+			setTimeout(() => {
+				updateHeights();
+			}, 100);
+
 			if (controlsRef.value) {
-				const controlsObserver = new ResizeObserver(updateContentHeight);
+				const controlsObserver = new ResizeObserver(debouncedUpdateHeights);
 				controlsObserver.observe(controlsRef.value);
 				onUnmounted(() => controlsObserver.disconnect());
 			}
-			window.addEventListener('resize', updateContentHeight);
-			onUnmounted(() => window.removeEventListener('resize', updateContentHeight));
+			if (headlineRef.value) {
+				const headlineObserver = new ResizeObserver(debouncedUpdateHeights);
+				headlineObserver.observe(headlineRef.value);
+				onUnmounted(() => headlineObserver.disconnect());
+			}
+			window.addEventListener('resize', debouncedUpdateHeights);
```

Comment:
In storybook I'm seeing a good height when the side sheet opens, but I'm not seeing the height update as the window size changes:
![Uploading image.png…]()